overlord/snapstate/handlers.go

```diff
@@ -3992,11 +3992,12 @@ func changeReadyUpToTask(task *state.Task) bool {
 // track of the instance names from the first SnapSetup in every lane, stopping
 // when finding the given task, and resetting things when finding a different
 // re-refresh task (that indicates the end of a batch that isn't the given one).
```

Comment:
the comment needs some updating to match the new code I think

Reply:
Done! Nice catch